pod/perlbook.pod

```diff
@@ -26,7 +26,7 @@ I<Programming Perl>:
  by Tom Christiansen, brian d foy, Larry Wall with Jon Orwant
  ISBN 978-0-596-00492-7 [4th edition February 2012]
  ISBN 978-1-4493-9890-3 [ebook]
- https://oreilly.com/catalog/9780596004927
+ https://www.oreilly.com/library/view/-/9781449321451/
```

Comment:
Some of the URLS in this document are encased in `L<>` POD tags; most are not.  Should we standardize?

Reply:
Since we don't use that URL but point to `https://www.learning-perl.com/` it shouldn't matter. The ISBN for the ebook is the same - `9781492094920`.

![image](https://github.com/Perl/perl5/assets/68062695/7d769c4c-ad53-4f4e-bb45-36153d65a20a)

Reply:
As for the links, I agree let's wrap them all in `L<>`.